bin/install.py

```diff
@@ -55,10 +58,10 @@
 
 def download_and_install(url, target):
     try:
-        urllib.urlretrieve(url, target)
-        os.chmod(target, 0755)
+        urllib.request.urlretrieve(url, target)
+        os.chmod(target, 0o755)
```

Comment:
`0o`? You sure?

Reply:
Yes (that's generated code, and it runs fine in the Python REPL)

Reply:
Let the Python REPL guide us!